### ISSUE TITLE
SAXBuilder.fileToURL method is not consistent with File.toURI().toURL()

### ISSUE THREAD

**Issue by rolfl** [hunterhacker/jdom#28]
fileToURL was seemingly writtent because the URL / URI classes were only introduced with Java 1.4.

There are some subtleties about URL's that this method does not accomodate. For example, it appears that for files, the 'authority' section for the file URL should (to be strictly compliant) be absent entirely (null), not "" (empty string), although most (all) processes that use file:/// type URLs will process the URL correctly.

For example, the file URL for the file /myfile should be: file:/myfile  ... but, JDOM produces file:///myfile

The reference for this appears to have been http://www.ietf.org/rfc/rfc2396.txt section 3, but this has been obsoleted by http://www.faqs.org/rfcs/rfc3986.html

RFC3986 clarifies the authority section of the URI in section 3.

The bottom line, in reading the RFC, is that file: URL's never have an authority section (because they can only refer to the local machine), and certainly, the authority host is not "". As a result, the correct URL is file:/path/to/file.

File.toURI().toURL() produces the correct results, and it seems logical to simply replace the SAXBuilder.fileToURL() with the standard 'Java' way of doing things, which will be better at following the changing standards.

I suggest just replaceing the fileToRUL method entirely.
